libkbfs/ops.go

```diff
@@ -190,8 +191,8 @@ func (co *createOp) String() string {
 	return res
 }
 
-func (co *createOp) CheckConflict(renamer ConflictRenamer, mergedOp op) (
-	crAction, error) {
+func (co *createOp) CheckConflict(renamer ConflictRenamer, mergedOp op,
```

Comment:
I can't think of any reason we'd want to treat executables different from regular files in anything other than `createOp.CheckConflict`, and in that case we already have the exact type in the ops themselves.  So I'm inclined to leave it unless you feel strongly.